### DISCUSSION TITLE
[Feature Request] [Status Bar] Drawn battery icon on PS3

### DISCUSSION THREAD

**Discussion by CorruptionVR** [EmiyaSyahriel/CrossLauncher#89] [Ideas]
I would like a way to put a drawn battery icon in the Status Bar on the PS3 XMB Layout. Make this optional, as users may be using the app on a TV, maybe as a key in the Status Bar Text Format dialogue.
ex: `{battery_icon}`

Clarification:
What I mean by drawn is referencing how it appears on the android status bar. As the battery percentage goes down, more of the icon becomes transparent and appears less full.

**Comment by EmiyaSyahriel** [comment]
I don't know if I can add this status format key or not, since the current implementation of status bar text only supports text and does not ~yet~ supports images/drawing except for Emoji, while the battery icon ( which is available on PSP Layout Mode ) is a custom drawing.

But I'll leave this open, maybe I am in the future or someone can look to it.

And also, what battery level does a TV actually refer to? Aren't they always plugged into main power source? is it Remocon battery level?
